### PR TITLE
DISTMONGO-200 Convert docs to Markdown (4.2)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
Configure MkDocs for doc build
Configure Render for docs hosting

### CODE REVIEW

**Review by nastena1606 — APPROVED** [review]

File: _resource/overrides/main.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+{# Import the theme's layout. #}
+{% extends "base.html" %}
+
+{%- macro relbar2 () %}
+<div class="md-relbar2__inner md-grid">
+<div class="md-content">
+  <article class="md-content__inner md-typeset" role="main">    
+    <h4>Contact Us </h4>
+       <p>For free technical help, visit the Percona <a class="reference external" href="https://forums.percona.com/c/postgresql/25?utm_campaign=Doc%20pages" target="_blank">Community Forum</a>.<br>
```
Comment (nastena1606) [inline _resource/overrides/main.html]: The URL must be the one for pdmdb:)

File: _resource/overrides/main.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+</div>
+</div>
+{%- endmacro %}
+
+              {% block content %}
+
+                <!-- Edit and view button -->
+                {% if page.edit_url %}
+                  {% set edit = "https://github.com/percona/pbm-docs/edit/main/docs/" %}
+                  {% set view = "https://raw.githubusercontent.com/percona/pbm-docs/main/docs/" %}
```
Comment (nastena1606) [inline _resource/overrides/main.html]: Let's change the source to pdmdb repo

File: mkdocs.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# MkDocs configuration for builds with material theme
+
+INHERIT: mkdocs-base.yml
+
+site_url: "https://docs.percona.com/percona-distribution-for-mongodb/"
```
Comment (nastena1606) [inline mkdocs.yml]: I believe we need the version here too

**Review by nastena1606 — COMMENTED** [review]
Let's remove the PDMDB-links.png as it refers to the Drupal layout

File: _resource/overrides/main.html (hunk 1 above)
Comment (alina-derkach-oaza) [reply _resource/overrides/main.html]: Updated

File: mkdocs.yml (hunk 3 above)
Comment (alina-derkach-oaza) [reply mkdocs.yml]: Updated

File: _resource/overrides/main.html (hunk 2 above)
Comment (alina-derkach-oaza) [reply _resource/overrides/main.html]: Updated

**Pull request merged**